### PR TITLE
Refactor/accessibility

### PR DESCRIPTION
**Pull request by Sulton88Mehron90**
What I did: refactor: Edit to html tag tag to comply with accessibility, 
                Edit to css property to comply with accessibility
                Edit to alt tag to be more descriptive of the img  property to comply with accessibility
                Add aria-label to html button tags.
Did this break anything?

- [x] No
- [ ]  Yes

Type of change

- [ ] New feature (non-breaking change which adds functionality)
- [ ]  Bug fix (non-breaking change which fixes an issue)
- [x]  Refactor(DRY-ing up/ reorganizing code, etc.)
- [ ]  Super small fix (Corrected a typo, removed a comment, etc.)
- [ ]  Skip all the other stuff and briefly explain the fix.

Checklist:

- [x]  If this code needs to be tested, all tests are passing
- [x]  The code runs locally
- [x]  There are comments where clarification/ organization is needed
- [x]  The code is DRY. If it's not, I tried my best
- [x]  I reviewed my code before pushing

What's next?

### CODE REVIEW

**Comment by Sulton88Mehron90** [comment]
refactor: Edit to html tag tag to comply with accessibility, 
                Edit to css property to comply with accessibility
                Edit to alt tag to be more descriptive of the img  property to comply with accessibility
                Add aria-label to html button tags.

File: dist/index.html [hunk 1]
```diff
@@ -12,19 +12,20 @@
   <body>
     <section class="home">
       <section class="home__header">
-        <div class="home__header2">
+        <div class="home__header__secondary"></div>
```
Comment (JamieCaudill) [inline dist/index.html]: Changing the class name here messes up the styling. Fix.

**Review by JamieCaudill — CHANGES_REQUESTED** [review]
Parvin. Good work on the alt text for images and additional ARIA tags. Please figure out why the header gets messed up with these changes and then send another PR.

File: dist/index.html (hunk 1 above)
Comment (Sulton88Mehron90) [reply dist/index.html]: i renamed it in css as well.

**Comment by Sulton88Mehron90** [comment]
ok will do.

File: dist/index.html (hunk 1 above)
Comment (Sulton88Mehron90) [reply dist/index.html]: refactoring in css for `.home__header2` effected the layouts  for home button and heading in a user page. It fixed the main page though 🤔 . So I just renamed the div back to `<div class="home__header2">`.
Consulted with my mentor he is saying that
Having two tags with different class names, such as `<div class="home__header"> `and `<div class="home__header2">`, is not inherently a bad practice for accessibility. Class names are primarily used for styling and selecting elements with CSS and JavaScript, so from an accessibility perspective, it doesn't directly impact the accessibility of the content. in future was advised not to use `header2` in the naming. 
Instead  something like `<div class="home__header__secondary` should be used.
the webpage still passing 100% on accessibility!

**Pull request merged**